scripts/tess/tess.py

```diff
@@ -0,0 +1,178 @@
+import datasets
+from datasets import Features, Value, Array2D, Sequence
+from datasets.data_files import DataFilesPatternsDict
+import itertools
+import h5py
+import numpy as np
+
+# TODO: Add BibTeX citation
+# Find for instance the citation on arxiv or on the dataset repo/website
+_CITATION = """\
+@ARTICLE{2020RNAAS...4..201C,
+       author = {{Caldwell}, Douglas A. and {Tenenbaum}, Peter and {Twicken}, Joseph D. and {Jenkins}, Jon M. and {Ting}, Eric and {Smith}, Jeffrey C. and {Hedges}, Christina and {Fausnaugh}, Michael M. and {Rose}, Mark and {Burke}, Christopher},
+        title = "{TESS Science Processing Operations Center FFI Target List Products}",
+      journal = {Research Notes of the American Astronomical Society},
+     keywords = {Catalogs, CCD photometry, Stellar photometry, 205, 208, 1620, Astrophysics - Earth and Planetary Astrophysics, Astrophysics - Instrumentation and Methods for Astrophysics, Astrophysics - Solar and Stellar Astrophysics},
+         year = 2020,
+        month = nov,
+       volume = {4},
+       number = {11},
+          eid = {201},
+        pages = {201},
+          doi = {10.3847/2515-5172/abc9b3},
+archivePrefix = {arXiv},
+       eprint = {2011.05495},
+ primaryClass = {astro-ph.EP},
+       adsurl = {https://ui.adsabs.harvard.edu/abs/2020RNAAS...4..201C},
+      adsnote = {Provided by the SAO/NASA Astrophysics Data System}
+}
+"""
+
+# TODO: Add description of the dataset here
+# You can copy an official description
+_DESCRIPTION = """\
+TESS Light Curves From Full Frame Images ("TESS-SPOC")
+"""
+
+# TODO: Add a link to an official homepage for the dataset here
+_HOMEPAGE = "https://archive.stsci.edu/hlsp/tess-spoc"
+
+# TODO: Add the licence for the dataset here if you can find it
+_LICENSE = "CC BY 4.0"
+
+_VERSION = "0.0.1"
+
+# Full list of features available here:
+# https://archive.stsci.edu/files/live/sites/mast/files/home/missions-and-data/active-missions/tess/_documents/EXP-TESS-ARC-ICD-TM-0014-Rev-F.pdf
+# https://cdsarc.cds.unistra.fr/ftp/IV/39/ReadMe
+#TODO: add additional features from the TIC catalog
+
+_FLOAT_FEATURES = [
+    "RA",
```

Comment:
no DEC? also is any other metadata, like object type available?

Reply:
I've added RA/DEC, it's also available directly as part of the light curve files (hadn't decided on the best way yet to include it). Object types are not available in one catalog unfortunately